tasks/create-snapshot-from-bundle-task.yaml

```
@@ -342,15 +348,25 @@ spec:
 
 
       def create_annotations(args):
-          return {
+          annotations = {}
+          # We want to keep the snapshot for tags that are not nightly tags.
+          regex = r"^refs/tags/(?!.*-nightly-).*"
+          if re.search(regex, args.target_branch):
```

[optional] `regex` can be just inlined:
```suggestion
          if re.search(r"^refs/tags/(?!.*-nightly-).*", args.target_branch):
```